.github/workflows/mssql-with-tbls.yml

```diff
@@ -0,0 +1,64 @@
+name: Microsoft SQL Server with tbls
+
+on:
+  pull_request:
+    paths:
+      - "samples/mssql-with-tbls/**"
+      - ".github/workflows/mssql-with-tbls.yml"
+  schedule:
+    # Schedule: Runs at 9:00 AM and 1:00 PM Asia/Tokyo time (UTC+9) on weekdays (Monday to Friday).
+    - cron: "0 0,4 * * mon-fri"
+
+jobs:
+  document:
+    runs-on: ubuntu-latest
+    services:
+      mssql:
+        image: mcr.microsoft.com/mssql/server:2022-latest
+        env:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: ${{ secrets.MSSQL_SA_PASSWORD }}
```

Comment:
No need to make it secrets because sample projects. Everything should be in plain text, including samples/mssql-with-tbls/.tbls.yml